test/test_valid_yaml.rb

```diff
@@ -17,6 +17,7 @@ def test_all_settings_in_base_yaml
 
     # Profiles using non-repo settings
     custom_profiles = %w[Chuno Erendel Shagium Crannach Dankmar Melborne Paeriluno Valkiss Ssarek Qetu Ugsy Valkiss Dartellum Leustyin Dijkstra Quillith]
+    shitty_settings = "knitting_tools"
```

Comment:
💡 Would it be easier just to fix each of the YAMLs currently in the repo?

Reply:
done now.